api/openapi-spec/swagger.json

```diff
@@ -4266,7 +4266,8 @@
           "$ref": "#/definitions/io.k8s.api.core.v1.LocalObjectReference"
         },
         "name": {
-          "type": "string"
+          "type": "string",
```

Comment:
I'm having trouble recreating this file. Which version of `openapi-gen` are you using? I am running against [release-1.19](https://github.com/kubernetes/kube-openapi) locally.

Reply:
How to check it? This is a bit of a strange tool. It was a bit of a struggle to install. There are no releases on github or aome "install" section. No `tags` in git either. And now that I finally have it running, there is no `version` or `--version` when calling `openapi-gen`.

@matthewmcnew Could you point me to how to install `openapi-gen 1.19`?

Reply:
But for now, I've reset `swagger.json` to `main`. There should be no change anyway, right?